packages/react-server-examples/hello-world/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "react-server-examples-helloworld",
+	"private": true,
+  "version": "0.0.1",
+  "description": "",
+  "main": "HelloWorld.js",
+  "scripts": {
+		"start": "react-server-cli",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Sasha Aickin",
+  "license": "Apache-2.0",
+  "dependencies": {
```

Comment:
Mind adding a reference to 'babel-runtime' here to make this truly plug and play?  Because of the routes file construction the error is consumed if installing this example onto a totally fresh machine.  And yes, I did not read the dependencies :)